pkg/providers/vault/secret_manager_test.go

```diff
@@ -109,12 +109,12 @@ func TestHttpSecretStoreManager_GetValue(t *testing.T) {
 			Data: testData,
 		}}
 
-	v, err := ssm.GetValue("one")
+	v, err := ssm.GetValues("one")
```

Comment:
Can we update the test functions name to match the function in the `SecretStoreManager`, basically pluralise. This goes for the other functions in this test file.